tutorials/demos.C

```diff
@@ -7,7 +7,7 @@
 ///
 /// \author Rene Brun
 
-{
+void demos() {
    gROOT->Reset();
```

Comment:
Could you also remove the call to `gROOT->Reset()`. please? It must not show up in named macros (and we don't want to give wrong examples in our tutorials).